mautic/mautic
# [UI/UX] Buttons

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

It changes buttons style and adds hierarchy.

![image](https://github.com/user-attachments/assets/cbe8c2f5-bd97-4077-bd2d-2e15fe877133)

![image](https://github.com/user-attachments/assets/f369d269-8d90-4ae0-b4d1-868ef2ea0046)

![image](https://github.com/user-attachments/assets/be9dc070-57f5-4e76-aa2b-030549637878)

![image](https://github.com/user-attachments/assets/7a48dc27-c658-4309-ba89-b8ba29991c05)

![image](https://github.com/user-attachments/assets/f668a628-a976-453c-8bf9-364a9286da87)

![image](https://github.com/user-attachments/assets/78171cd7-98ab-4ce6-82db-cbe13b6801e8)

![image](https://github.com/user-attachments/assets/e8ad62c7-8d0d-4c27-b66c-df6b4fee0ee0)

Specifically:
- Adds icons for campaign builder buttons
- Define primary, secondary, tertiary, ghost and icon buttons






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the dashboard > Look at the buttons
3. Open Contacts to see a scenario where we have 2 primary buttons (Quick add and New, on the technical side), but the second (new) and the dropdown become tertiary automatically
4. Create a new contact, Save & Close
5. Open Campaigns > New > Builder > See buttons with icons
